app/services/invoices/advance_charges_service.rb

```diff
@@ -79,7 +79,16 @@ def create_group_invoice
           raise ActiveRecord::Rollback
         end
 
-        Invoices::ComputeAmountsFromFees.call(invoice:)
+        # NOTE: We don't want to use Invoices::ComputeAmountsFromFees here
+        #       because it would recompute taxes from pre-tax values. All Fees are already paid
+        #       this invoice should show how much taxes were paid in total.
+        # NOTE: progressing billing, coupons and credit notes are not supported here
+        invoice.fees_amount_cents = invoice.fees.sum(&:amount_cents)
```

Comment:
Just one though, should we take coupons amount into account as well?

Reply:
I'll make sure Matt and the customer are aware of it. If we want to take care of it, I'd do it in a separate PR because I'll need some more spec scenario to understand 😬